src/Interpreters/CacheLog.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <Core/NamesAndAliases.h>
+#include <Core/NamesAndTypes.h>
+#include <Core/Settings.h>
+#include <DataTypes/DataTypeArray.h>
+#include <DataTypes/DataTypeTuple.h>
+#include <Interpreters/SystemLog.h>
+#include <Interpreters/TransactionVersionMetadata.h>
+#include <base/logger_useful.h>
+
+namespace DB
+{
+struct CacheLogElement
```

Comment:
```suggestion
struct FilesystemCacheLogElement
```
Let's do this in all similar places.